test-data/unit/check-inference.test

```diff
@@ -1959,7 +1959,7 @@ class C:
 
 if bool():
     f()
-    1 + '' # E: Unsupported left operand type for + ("int")
+    1 + '' # E: Unsupported operand types for + ("int" and "str")
```

Comment:
I'd prefer changing the test case so that we don't depend on `int.__add__` instead (it's a little arbitrary to rely on it here). For example, use `1()` instead of `1 + ''` which doesn't require a fixture change.